ui/src/message_popup/popup_question.jsx

```diff
@@ -29,9 +29,9 @@ export default React.createClass({
   mixins: [SetIntervalMixin],
 
   propTypes: {
-    shouldShowStudentCard: React.PropTypes.bool.isRequired,
-    shouldShowSummary: React.PropTypes.bool.isRequired,
-    helpType: React.PropTypes.string.isRequired,
+    scaffolding: React.PropTypes.object.isRequired,
```

Comment:
You can also use `React.PropTypes.shape` to express that scaffolding needs to have a few fields (and to have Flow check that those are the only ones we're accessing).